hw_Kutovyi.py

```diff
@@ -0,0 +1,138 @@
+from typing import List
+from pydantic import BaseModel
+
+
+class BookModel(BaseModel):
+    title: str
+    author: str
+    year: int
+
+    def __str__(self):
+        return f"{self.title} ({self.author}, {self.year})"
+
+
+
+class Library:
+    def __init__(self):
+        self.books: List[BookModel] = []
+
+    def add_book(self, book: BookModel):
+        self.books.append(book)
+
+    def remove_book(self, title: str):
+        self.books = [book for book in self.books if book.title != title]
+
+    def list_books(self):
+        for book in self.books:
+            print(book)
```

Comment:
You can do it similar with list comprehension like you did with remove_book method